testdata/src/common/common.go.golden

```diff
@@ -4,18 +4,18 @@ import (
 	"net/http"
 )
 
-const testHeader = "testHeader"
+const constTestHeader = "testHeaderValue"
 
 func p() {
 	v := http.Header{}
-	v.Get(testHeader) // want `const "testHeader" used as a key at http.Header, but "testHeader" is not canonical, want "Testheader"`
+	v.Get(constTestHeader) // want `use "Testheadervalue" instead of "testHeaderValue"`
```

Comment:
The previous code was reporting it was a constant, here the error message is strange

Reply:
I think it's more simple.

There are fewer words here and more robotic.

You can understand what the problem is by taking a quick look

Reply:
I liked the previous way, but why not

Reply:
I'd rather keep it as simple as it is rn. Mentioning a constant might make an impression that the linter insist on renaming the constant, instead of changing its value or using some other value

Reply:
@ccoVeille, can I close this conversation?